### DISCUSSION TITLE
Gridpoint API pressure units / conversion

### DISCUSSION THREAD

**Discussion by oisact** [weather-gov/api#250] [Q&A]
In the gridpoint forecast API (eg https://api.weather.gov/gridpoints/RNK/34,49) the `pressure` values do not specify a unit.  These appear to be in `inHg`, however the values are far too low.  After testing various scenarios, these values appear to represent the pressure at the elevation above sea level of that station.  In other words, to make use of these numbers at surface-level you would have to convert them to sea-level elevation.

That seems strange, and I cannot find any documentation to that effect.  If that is indeed the case, what is the formula to use to convert the pressure to the standard surface-level value?  Are we to take into consideration the temperature and elevation or just the elevation? Is this a bug or oversight that the barometric pressure is provided in such a strange (to me) way?

**Comment by jfbauer432** [comment]
I have had the same question myself.  I even tried a few different adjustment to sea level formulas, but nothing I tried seemed to match what was expected.
